### PR TITLE
Changes made to newgame.jade, moneyTracker.js

### PR DESCRIPTION
**Pull request by jboretsky**
also now I should be committing as jboretsky instead of Heather... I hadn't set up a username from my command line so i think it was just using the default username on the computer

### CODE REVIEW

File: views/index.jade [hunk 1]
```diff
@@ -3,8 +3,7 @@ extends layout
 block content
     div(class="container-fluid", ng-hide="clicked")
         .row
-            div(class="col-sm-3 col-md-4 col-lg-4" )
-            div(class="col-sm-6 col-md-4 col-lg-4" )
+            .col-sm-6.col-md-4.col-lg-4.col-sm-offset-3.col-md-offset-4.col-lg-offset-4
```
Comment (andreirtaylor) [inline views/index.jade:6]: Yeah this was a messy hack, im glad that you stripped it out.

File: public/javascripts/moneyTracker.js [hunk 2]
```diff
@@ -163,7 +168,12 @@ app.run(function(editableOptions) {
                 }
             };
             $scope.removePlayer = function(name){
-                $scope.playerList.splice(name,1);
```
Comment (andreirtaylor) [inline public/javascripts/moneyTracker.js:166]: Im surprised we didnt catch this sooner. Simple mistake due to javascript never wanting to error.

**Comment by andreirtaylor** [comment]
I think it looks ok. You want to merge now or after we get the red flash working?

**Comment by jboretsky** [comment]
merge now, I haven't pushed the newgame.jade file yet so this is all finished stuff.

**Pull request merged**